### PR TITLE
Feature/number of patients chosing every doctor

### PR DESCRIPTION
**Pull request by kkrizan75**

### CODE REVIEW

**Comment by kkrizan75** [comment]
image
image

File: src/app/manager/allergen-statistics-frequency/allergen-statistics-frequency.component.ts [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    options: {
+      plugins: {
+        legend: {
+            labels: {
+                // This more specific font property overrides the global property
+                font: {
+                    size: 20
+                }
+            }
+        }
```
Comment (JanosevicRa177) [inline src/app/manager/allergen-statistics-frequency/allergen-statistics-frequency.component.ts]: refactor braces

File: src/app/manager/allergen-statistics-frequency/allergen-statistics-frequency.component.ts [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+      }
+    }
+    });
+  }
+
+  ngOnInit(): void {
+    this.allergenStatsService.getAllergensWithPatients().subscribe(res =>{
+      this.allergensWithPatients = res;
+      console.log(res);
+      this.allergensWithPatients.forEach( allergenWithPats => {
```
Comment (JanosevicRa177) [inline src/app/manager/allergen-statistics-frequency/allergen-statistics-frequency.component.ts]: delete allergensWithPatients(isnt needed) and delete console.log

File: src/app/manager/doctor-statistics-popularity/doctor-statistics-popularity.component.ts [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+@Component({
+  selector: 'app-doctor-statistics-popularity',
+  templateUrl: './doctor-statistics-popularity.component.html',
+  styleUrls: ['./doctor-statistics-popularity.component.css']
+})
+
+
+
+
```
Comment (JanosevicRa177) [inline src/app/manager/doctor-statistics-popularity/doctor-statistics-popularity.component.ts]: delete unnecessary

File: src/app/manager/doctor-statistics-popularity/doctor-statistics-popularity.component.ts [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+        this.doctors.push(doctorWithPopularity.id + ' ' + doctorWithPopularity.name + ' ' + doctorWithPopularity.surname);
+        this.doctorsPopularities.push(doctorWithPopularity.patientsPicked);
+      });
+    this.createChart();
+  });
+  }
+
+
+
+
```
Comment (JanosevicRa177) [inline src/app/manager/doctor-statistics-popularity/doctor-statistics-popularity.component.ts]: delete unnecessary spaces

File: src/app/manager/doctor-statistics-popularity/doctor-statistics-popularity.component.ts [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+            callback: function (value) { if (Number.isInteger(value)) { return value; } return null;},
+          },
+          beginAtZero: true
+        }
+      }
+    }
+    });
+    this.doctors = [];
+    this.doctorsPopularities = [];
+  }
```
Comment (JanosevicRa177) [inline src/app/manager/doctor-statistics-popularity/doctor-statistics-popularity.component.ts]: refactor braces

File: src/app/manager/doctor-statistics-popularity/doctor-statistics-popularity.component.ts [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+  });
+  }
+
+
+
+
+  async ngOnInit(): Promise<void> {
+
+    this.dcotorStatisticsPopularityService.getDoctorsWithPopularity(0,666).subscribe(res => {
+      this.doctorsWithPopularity = res;
```
Comment (JanosevicRa177) [inline src/app/manager/doctor-statistics-popularity/doctor-statistics-popularity.component.ts]: remove unnecessary property

File: src/app/manager/doctor-statistics-popularity/doctor-statistics-popularity.component.ts [hunk 7]
```diff
@@ -0,0 +1,114 @@
...
+
+     this.doctorsWithPopularity.forEach( (doctorWithPopularity:IDoctorWithPopularityDTO,index) => {
+        this.doctors.push(doctorWithPopularity.id + ' ' + doctorWithPopularity.name + ' ' + doctorWithPopularity.surname);
+        this.doctorsPopularities.push(doctorWithPopularity.patientsPicked);
+      });
+      this.createChart();
+  });
```
Comment (JanosevicRa177) [inline src/app/manager/doctor-statistics-popularity/doctor-statistics-popularity.component.ts]: just call function you made

**Review by JanosevicRa177 — CHANGES_REQUESTED** [review]

**Review by JanosevicRa177 — APPROVED** [review]

**Pull request merged**